content/doc/book/using/best-practices.adoc

```diff
@@ -0,0 +1,177 @@
+---
+layout: section
+title: Best Practices
+---
+ifdef::backend-html5[]
+:description:
+:author:
+:sectanchors:
+:toc:
+:toclevels: 4
+:hide-uri-scheme:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+endif::[]
+
+Continuous Integration (CI) with automated test execution and trends has changed the way companies look at build management, release management, deployment automation, and test orchestration.
+This section describes best practices to provide executives, business managers, software developers, and architects with a better sense of what Jenkins can contribute throughout the project lifecycle.
+
+== Automate job definition
+
+Jenkins can automatically create, update, and delete jobs based on the repositories it detects in your software configuration management system.
+Structure your job definitions to get the most benefit from automatic job management by Jenkins.
+
+There are multiple alternatives for automatic job management, including:
+
+* <<Use organization folders>> - create, update, and delete multibranch Pipeline folders and Pipeline jobs automatically (preferred)
+* <<Use multibranch Pipelines>> - create, update and deleted Pipeline jobs automatically
+* <<Use Pipeline>> - manually defined Pipeline jobs
+
+=== Use organization folders
+
+Organization Folders offer a convenient way to allow Jenkins to automatically create and delete jobs as repositories and branches are created and deleted.
+If you use GitHub organizations, Bitbucket teams, GitLab groups, or Gitea organizations, then Jenkins can automatically detect the creation of a new repository and create a Multibranch Pipeline project for it.
+
+Refer to the link:/doc/book/pipeline/pipeline-as-code/#organization-folders[organization folders] documentation for more details.
+
+.GitHub organization folders
+video::LbXKUKQ24T8[youtube,width=800,height=420]
+
+.GitLab group folders
+video::it6TOeQ6EHg[youtube,width=800,height=420]
+
+.Bitbucket project folders
+video::85b6fiVolfk[youtube,width=800,height=420]
+
+.Gitea organization folders
+video::NO3sZWRxgQM[youtube,width=800,height=420]
+
+=== Use multibranch Pipelines
+
+If something prevents you from using organization folders, then use multibranch Pipelines.
+Organization folders are preferred over multibranch Pipelines because they will automatically create and delete multibranch projects when repositories are created or deleted.
```

Comment:
```suggestion
However, it's important to note that organization folders are preferred over multibranch Pipelines because they provide the automation of creating and deleting multibranch projects when repositories are added or removed.
```